pkg/maps/policymap/policymap.go

```diff
@@ -198,9 +199,9 @@ func (pm *PolicyMap) Exists(id uint32, dport uint16, proto u8proto.U8proto, traf
 
 // DeleteKey deletes the key-value pair from the given PolicyMap with PolicyKey
 // k. Returns an error if deletion from the PolicyMap fails.
-func (pm *PolicyMap) DeleteKey(key PolicyKey) error {
+func (pm *PolicyMap) DeleteKeyWithErrno(key PolicyKey) (error, syscall.Errno) {
```

Comment:
Make this a new function instead of overwriting the old one.

Reply:
Actually, nevermind, this is the only user of this function, let's just keep it as you have done it.